index.html

```diff
@@ -20,6 +20,7 @@
     <link rel="stylesheet" type="text/css" href="/colors.css" />
     <link rel="stylesheet" type="text/css" href="/elevations.css" />
     <link rel="stylesheet" type="text/css" href="/layout.css" />
+    <link rel="stylesheet" type="text/css" href="/code.css" />
```

Comment:
Could we call this `prettylights.css`?

Reply:
aba5300f8ada40e03e5f5d8b917f8d71bd115e05